app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
-          numbers are only an example, yours may be different).
+          In the dropdown menu in the top left of this program, there is a list of IP addresses and a port number. If there is more than 1 IP address,
+          you must determine which one is the IP address that talks on the same network as your phone. These numbers are what you will be entering
+          into your phone (these numbers are only an example, yours may be different).
         </p>
         <Image src="../assets/help_network_settings.png" bordered />
-        <h3>Android</h3>
-        <ol>
-          <li>Open Settings and select Wi-Fi</li>
-          <li>Long press on your connected Wi-Fi network and select Modify Network</li>
-          <li>Expand Advanced Options</li>
-          <li>Set Proxy to Manual</li>
-          <li>
-            Enter your computer's IP address in the <kbd>Proxy hostname</kbd> field
-          </li>
-          <li>
-            Enter the port number in the <kbd>Proxy port</kbd> field
-          </li>
-          <li>Save changes and exit settings</li>
-        </ol>
-        <h3>iOS</h3>
-        <ol>
-          <li>Open Settings and select Wi-Fi</li>
-          <li>Tap the Wi-Fi network you are currently connected to</li>
-          <li>
-            Scroll down to <kbd>HTTP Proxy</kbd> and click Manual
-          </li>
-          <li>
-            Enter your computer's IP address in the <kbd>Server</kbd> field
-          </li>
-          <li>
-            Enter the port number in the <kbd>Port</kbd> field
-          </li>
-          <li>
-            Leave Authentication <kbd>OFF</kbd>
-          </li>
-          <li>Save changes and exit settings</li>
-        </ol>
+        <h3>Setup Instructions step by step</h3>
+        <Accordion>
+          <Accordion.Title active={activeIndex === 0} index={0} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Limitations [READ THIS FIRST]
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 0}>
+            SWEX won't work for unrooted Android 7+. The reason is that in Android 7+, apps don't accept user signed certs by default. App devs would
+            need to allow them specifically. For people that use Android 7+, you need to use Emulators like Mumu or Nox which usually run older
+            Android versions. Device with Android 6 or lower also work. iOS does not have this limitation and works fine in all versions if you do the
+            additional steps. Using BlueStacks is not reconmended but may be possible, although the last confirmed working verions for this is 0.0.29
+            which doesn't support the needed HTTPS functionality. This also requires root access. We are not liable for any form of damage caused by
+            using any BlueStacks setup guide.
+          </Accordion.Content>
+
+          {/* Documents & programs */}
```

Comment:
Not a huge fan of these meaningless comments, since you can see the section directly under it anyway. Would suggest to remove these.